src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int GROW_FACTOR = 2;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+
+    private List<Entry<K, V>>[] buckets;
```

Comment:
Better use usual array instead of List

Reply:
I have tried these changes and they only clutter the code and require additional logic as an array provides less possibilities than a list. Also, the list under the hood is an array and the current solution does not complicate or worsen the work of the map in any way